.github/workflows/ci_run_pubsub_electron_cfg.json

```diff
@@ -0,0 +1,13 @@
+{
+    "language": "Javascript",
+    "runnable_file": "./aws-iot-device-sdk-js-v2/samples/node/pub_sub_electron_node",
+    "runnable_region": "us-east-1",
+    "runnable_main_class": "",
+    "arguments": [
+        {
+            "data": "is_ci"
+        }
+    ],
+    "node_cmd": "npm start",
+    "timeout": 180
```

Comment:
Electron windows sample would failed with a error prompt, and hangs in the CI. Set a timeout here to fail the process in this case.